codenames/utils/file_utils.py

```diff
@@ -0,0 +1,7 @@
+from typing import List
```

Comment:
May be merge this with `codenames/util.py` or move that file into `codenames/utils`, and call it `game_utils.py` or something?

Reply:
Done.